src/node_buffer.cc

```diff
@@ -1142,6 +1142,143 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
                                 : -1);
 }
 
+void NoopCallback(char* data, void* hint) {
+  // Intentional no-op
+}
+
+void Split(const FunctionCallbackInfo<Value>& args) {
```

Comment:
This should be leveraging a lot of what's in `Buffer::IndexOfBuffer()`. Would simply this and keep implementations unified.